ONSdigital/eq-questionnaire-launcher
# Fix url schema launch

### What is the context of this PR?
This makes the change to only send the schema url to runner when launching via a schema url

Also fixes the alignment of the arrow icon next to the load previous value button and tidys up the code a bit.

### How to review
- Run this launcher branch locally, see that the arrow icon is now aligned
- Launch a schema from a url and see that the schema launches. This is a url you can use to test that: `https://raw.githubusercontent.com/ONSdigital/eq-questionnaire-schemas/main/schemas/business/en/1_0005.json`
- Use the quick launch facility to launch a schema and see that the schema launches correctly without error. This is a url you can use to test that: `http://localhost:8000/quick-launch?url=https://raw.githubusercontent.com/ONSdigital/eq-questionnaire-schemas/main/schemas/business/en/1_0005.json`
